### ISSUE TITLE
"Running problem"

### ISSUE THREAD

**Issue by lsyfffighting** [xhan97/StreaKHC#2]
Hello, can the datasets in the 'runned' folder be used for testing as well?

**Comment by xhan97** [comment]
Yes, they all can be used for evaluation.  But you need to manually move them to the ‘’raw“ directory if you do not want to change the code.
